### DISCUSSION TITLE
Pi Juice Gpio

### DISCUSSION THREAD

**Discussion by gencosystems** [PiSupply/PiJuice#1007] [General]
Can somebody kindly tell me exactly which GPIO pins are used by Pi Juice when connected to a raspberry pi please.

Many thanks
Graham

**Comment by tvoverbeek** [comment]
The I2C pins (pins 3 and 5) and the power pins (5V and GND).
See also the PiJuice schematic

**Reply by gencosystems** [reply]
Hi Ton
Very many thanks.

Kind regards
Graham

**Reply by gencosystems** [reply]
So four (4) pins
